src/utils/NativeTokenRecurringAllowance.sol

```diff
@@ -60,12 +67,8 @@ abstract contract NativeTokenRecurringAllowance {
     ///
     /// @param account Account that spent native token via a permission.
     /// @param permissionHash Hash of the permission.
-    /// @param cycleStart Start time of the cycle this spend was used on (unix seconds).
-    /// @param cycleEnd End time of the cycle this spend was used on (unix seconds).
-    /// @param spend Amount of native token spent (wei).
-    event RecurringAllowanceUsed(
-        address indexed account, bytes32 indexed permissionHash, uint48 cycleStart, uint48 cycleEnd, uint256 spend
-    );
+    /// @param newUsage Start and end of the current cycle with new spend usage (struct).
+    event RecurringAllowanceUsed(address indexed account, bytes32 indexed permissionHash, CycleUsage newUsage);
```

Comment:
Refactor event arg to reuse struct for cleanliness and parity with `RecurringAllowanceInitialized`